src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -1,129 +1,140 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
what happened to this file? space issues?

Reply:
It's due to merge from master branch.